### PR TITLE
Api 3998 turn off sending uploads to gdit

### PR DESCRIPTION
**Pull request by gpbowman-git**
## Description of change

Commenting out jobs that push uploads to GDIT starting on 12/10/2020 during a transition from GDIT to GovCIO for central mail.

## Original issue(s)
https://vajira.max.gov/browse/API-3998

### CODE REVIEW

File: modules/vba_documents/app/controllers/vba_documents/v1/uploads_controller.rb [hunk 1]
```diff
@@ -34,7 +34,8 @@ def show
           submission.status = request.headers['Status-Override']
           submission.save
         else
-          submission.refresh_status! unless submission.status == 'expired'
+          Rails.logger.info 'remove me! The linter wants a line here'
```
Comment (LindseySaari) [inline modules/vba_documents/app/controllers/vba_documents/v1/uploads_controller.rb:37]: Is this PR ready for review or are you still debugging? Just noticed this line

**Comment by LindseySaari** [comment]
Do you plan on reverting these changes at some point after the transition? I would recommend even deleting the code instead of commenting it out. If a revert is in the plans, it might be good to link to the ticket for that work in the PR description also

File: modules/vba_documents/app/controllers/vba_documents/v1/uploads_controller.rb (hunk 1 above)
Comment (gpbowman-git) [reply modules/vba_documents/app/controllers/vba_documents/v1/uploads_controller.rb]: This is ready for review. These changes will be rolled back once the transition to GovCIO is complete.

**Review by bastosmichael — APPROVED** [review]
Yes our goal is to revert this completely once the GDIT to GCIO transition is complete, I do like the idea of removing instead of commenting out but for time purposes I'll approve this as is for now.

**Pull request merged**